.drone.yml

```diff
@@ -1,32 +1,55 @@
+---
 kind: pipeline
-type: docker
+type: kubernetes
 name: default
 
-
 steps:
   - name: test
-    image: golang:1.13
+    image: golang:1.13-buster
     commands:
-      - go test ./cmd/... ./internal/...
-      - go vet ./cmd/... ./internal/...
+      - make test
+
   - name: lint
     image: cytopia/golint
     commands:
-      - golint -set_exit_status ./cmd/... ./internal/...
-  - name: build
-    image: golang:1.13
-    commands:
-      - go build -o build/drone-helm cmd/drone-helm/main.go
+      - golint -set_exit_status ./...
+
   - name: publish_linux_amd64
     image: plugins/docker
     settings:
-      auto_tag: true
+      registry: quay.io
+      repo: quay.io/mongodb/drone-helm
```

Comment:
Be sure to disable the drone repo for http://github.com/mongodb-forks/drone-helm